pages/dkp/konvoy/1.8/release-notes/1.8.4/index.md

```diff
@@ -0,0 +1,75 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes Konvoy 1.8.4
+title: Release Notes Konvoy 1.8.4
+menuWeight: 50
+excerpt: View release-specific information for Konvoy 1.8.4
+beta: false
+enterprise: false
+---
+<!-- markdownlint-disable MD034 -->
+
+**D2iQ&reg; Konvoy&reg; version 1.8.4 was released on **
+
+[button color="purple" href="https://support.d2iq.com/hc/en-us/articles/4409215222932-Product-Downloads"]Download Konvoy[/button]
+
+<p class="message--note"><strong>NOTE: </strong>You must be a registered user and logged on to the support portal to download this product. For new customers, contact your sales representative or <a href="mailto:sales@d2iq.com">sales@d2iq.com</a> before attempting to download Konvoy.</p>
+
+## Release summary
+
+This release provides new features and enhancements to improve the user experience, fix reported issues, integrate changes from previous releases, and maintain compatibility and support for other packages used in Konvoy.
+
+### Supported versions
+
+| Kubernetes Support | Version |
+| ------------------ | ------- |
+| **Minimum**  | 1.18.x  |
+| **Maximum**  | 1.20.x  |
+| **Default**  | 1.20.13  |
+
+### New features and capabilities
+
+#### Networking
+
+- Calico has been updated to v3.20.2 to address an issue that prevented container deletion. (COPS-7092)
+
+#### Autoscaling
+
+- Fixed an issue installing the auto-provisioning component in an Ubuntu environment. (COPS-7102)
+
+#### Air-Gapped
+
+- RHEL 8 FIPS packages have been updated to produce the necessary SHA256 signatures during installation. (COPS-7066)
+
+### Bug Fixes
+
+#### Authentication
+
+- Users within the system:authenticated group are now able to access group-appropriate URLs. (COPS-7067)
+
+#### Prometheus
+
+-  Prometheus addon upgrades now appropriately upgrade the alertmanager CRD.
```

Comment:
```suggestion
### Known Issues

-  Prometheus addon upgrades fail due to a missing alertmanager CRD. To work around this issue, manually apply the missing CRD to the cluster prior to upgrading by running: `kubectl apply -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/v0.47.0/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml`
```

Let's add the workaround for the cops ticket since it won't make it into this patch reelase: https://jira.d2iq.com/browse/COPS-6842